### PR TITLE
Add recurring session command -UG

### PR DESCRIPTION
**Pull request by nowknowing**
Add command for recurring session

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -178,6 +178,22 @@ Format: `add_session n/STUDENT_NAME d/DATE t/TIME k/DURATION s/SUBJECT f/FEE`
 Examples:
 * `add_session n/John Doe d/2021-01-01 t/18:00 k/120 s/Biology f/80`
 
+### Adding a recurring tuition session: `add_rec_session`
```
Comment (enhao25) [inline docs/UserGuide.md:284]: Looks good, however, do remember to add this command over to the TOC and the command summary at the end of the page for consistency.

**Review by enhao25 — COMMENTED** [review]
Some comments for consistency.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -259,9 +262,9 @@ The command `list_session` will show the following:
 * `find_session Zach` returns nothing-->
 <!-- END OF COMMENT OUT FOR FIND SESSION -->
 
-### Adding a tuition session: `add_session`
+### Adding a single tuition session: `add_session`
 
-Adds a tuition session to the TutorBuddy
+Adds a single tuition session to the TutorBuddy
```
Comment (enhao25) [inline docs/UserGuide.md:267]: Hi, just wondering, is "to the TutorBuddy" quite werid, I tink its copied over from single session also. I think changing over to "to TutorBuddy" removing "the" would be better. What do you think?

**Review by enhao25 — APPROVED** [review]
LGTM, just some minor english comments.

File: docs/UserGuide.md (hunk 2 above)
Comment (JonahhGohh) [reply docs/UserGuide.md:267]: Sounds good. I'll change it on my end.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -278,6 +281,22 @@ TutorBuddy takes care of overlapping session for you by giving a gentle prompt,
...
+Similar to arguments for adding a single tuition session, except:
+* `DATE` refers to the starting date of the session
+
+Arguments additional to adding a single tuition session:  
+* `INTERVAL` should be in days, including the date from and excluding the date to <br/>
+  e.g. `INTERVAL` = 7 for weekly sessions
+* `ENDDATE` should be an ending date of the recurring session, consistent with `DATE` and `INTERVAL`
+
+Examples:
+* `add_session n/John Doe d/2021-01-01 t/18:00 k/120 s/Biology f/80 b/7 e/2021-01-29`
```
Comment (JonahhGohh) [inline docs/UserGuide.md:299]: Here the command should be add_recurring_session?

**Review by JonahhGohh — APPROVED** [review]
LGTM, I'll merge first and make the necessary change on my end.

**Pull request merged**